test/integration/options.spec.js

```diff
@@ -180,4 +183,28 @@ describe('options', function () {
       });
     });
   });
+
+  describe('--help', function () {
+    it('works despite the presence of mocha.opts', function (done) {
+      var output = '';
+      try {
+        fs.mkdirSync(path.resolve(__dirname, 'test-env'));
+        fs.mkdirSync(path.resolve(__dirname, 'test-env/test'));
+        fs.writeFileSync(path.resolve(__dirname, 'test-env/test/mocha.opts'), 'foo');
+        var mochaLoc = path.resolve(__dirname, '../../bin/mocha');
+        output = '' + childProcess.execSync(mochaLoc + ' -h', {cwd: path.resolve(__dirname, 'test-env')});
+      } catch (e) {}
+      try {
```

Comment:
The sequence of `try...catch` blocks here are for setup and teardown, and should be shifted to `before` and `after` blocks, outside this `it` block.